client/src/types/material-ui.d.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Think you can delete this? Both imports are used

Reply:
they're actually not, we are creating our own types with the same names
(which i'm not sure if this is a good thing, but a bunch of other things break if we remove this module declaration or if we remove these imports 🤷, fixing either of those will have to come in another PR)

if you remove my comment and run `npm run lint` from client folder, you should see warnings. lmk if not

Reply:
Yep, that gave warnings. Interesting!